packages/react-dom/src/__tests__/ReactDOMFragmentRefs-test.js

```diff
@@ -884,4 +884,80 @@ describe('FragmentRefs', () => {
       expect(clientRects[2].left).toBe(9);
     });
   });
+
+  describe('getRootNode', () => {
+    // @gate enableFragmentRefs
+    it('returns the root node of the parent', async () => {
+      const fragmentRef = React.createRef();
+      const root = ReactDOMClient.createRoot(container);
+
+      function Test() {
+        return (
+          <div>
+            <React.Fragment ref={fragmentRef}>
+              <div />
+            </React.Fragment>
+          </div>
+        );
+      }
+
+      await act(() => root.render(<Test />));
+      expect(fragmentRef.current.getRootNode()).toBe(document);
+    });
+
+    // @gate enableFragmentRefs
+    it('returns the topmost disconnected element if the fragment and parent are unmounted', async () => {
```

Comment:
The assertion here fails because the fragment's `return` is `null` after unmounting it with its parent